### PR TITLE
Default pretraining gradient clipping & bias/norm weight decay filtering

### PR DESCRIPTION
**Pull request by warner-benjamin**
This PR turns on gradient clipping at a conservative value of 1.0 and enables bias & normalization weight decay filtering for pretraining by default.

I also renamed the weight decay filter option name to hopefully be a bit more self-explanatory.

### CODE REVIEW

**Comment by NohTow** [comment]
LGTM

I don't know if anyone has strong opinion about bias/norm WD filtering, but I think it's ok

**Comment by bclavie** [comment]
LGTM, no strong opinions.
